cloud.startup.hook/src/main/java/de/valtech/aecu/startuphook/AecuCloudStartupService.java

```diff
@@ -140,7 +140,9 @@ protected boolean waitForServices() throws InterruptedException {
      * Checks if our services are already injected.
      */
     private boolean servicesAreOk() {
-        Bundle bundle = FrameworkUtil.getBundle(BindingExtensionProvider.class);
+        Bundle bundle = Arrays.stream(bundleContext.getBundles())
+                              .filter(b -> GROOVY_CONSOLE_BUNDLE_SYMBOLIC_NAME.equals(b.getSymbolicName()))
+                              .findFirst().orElse(null);
```

Comment:
Hi @rubnig 

Solution looks good to me, thanks a lot. I haven't tested yet on a cloud env, I guess you did as you also noticed?

Small one which was already present before your solution: let's add a null check for the the bundle before using it further.
```
if (bundle == null) {
   return false;
}
```

Reply:
Yes we've encountered this issue on a Cloud environment and I've ran the above code via the Groovy Console on a Cloud environment, to check if it works.

Added the null check to avoid NPE ;)